src/main/resources/avro/MAPRecord.avsc

```diff
@@ -1079,6 +1079,36 @@
         "null",
         "string"
       ]
+    },
+  {
+    "name":
+    "messages"
+    ,
+    "type": {
+      "name": "MessagesIngestMessageArray",
+      "type": "array",
+      "items": {
+      "name": "MessagesIngestMessage",
+      "type": "record",
+      "fields": [ {
+      "name": "message",
+      "type": "string"
+    }, {
+      "name": "level",
```

Comment:
Avro has Enums which you might want to consider. Will save space and constrain the level to a specific range of values.

Reply:
Should have something on this by the end of the day

Reply:
This is addressed in #181